interchain.go

```diff
@@ -137,6 +137,13 @@ type InterchainBuildOptions struct {
 	// but it does still configure keys and wallets for declared relayer-chain links.
 	// This is useful for tests that need lower-level access to configuring relayers.
 	SkipPathCreation bool
+
+	// Optional. Git sha for test invocation. Once Go 1.18 supported,
+	// may be deprecated in favor of runtime/debug.ReadBuildInfo.
+	GitSha string
+
+	// If set, saves block history to a sqlite3 database to aid debugging.
+	BlocksDatabaseFile string
```

Comment:
Can this field be named `BlocksDatabasePath` since it is a path, not a file object?